specs/scoring_spec.rb

```diff
@@ -25,38 +28,60 @@
       Scrabble::Scoring.score('DoG').must_equal 5
     end
 
-    it 'returns nil for strings containing bad characters' do
-      Scrabble::Scoring.score('#$%^').must_be_nil
-      Scrabble::Scoring.score('char^').must_be_nil
-      Scrabble::Scoring.score(' ').must_be_nil
+    it 'throw ArgumentError for non-string inputs' do
+      proc { Scrabble::Scoring.score([]) }.must_raise ArgumentError
+      proc { Scrabble::Scoring.score(nil) }.must_raise ArgumentError
+    end
+
+    it 'throw ArgumentError inputs that non-alphabet characters' do
+      proc { Scrabble::Scoring.score('c4ts') }.must_raise ArgumentError
+      proc { Scrabble::Scoring.score('#$%^') }.must_raise ArgumentError
+      proc { Scrabble::Scoring.score('char^') }.must_raise ArgumentError
     end
 
-    it 'returns nil for words > 7 letters' do
-      Scrabble::Scoring.score('abcdefgh').must_be_nil
+    it 'throw ArgumentError for words > 7 letters' do
+      proc { Scrabble::Scoring.score('academies') }.must_raise ArgumentError
     end
 
-    it 'returns nil for empty words' do
-      Scrabble::Scoring.score('').must_be_nil
+    it 'throw ArgumentError for empty words' do
+      proc { Scrabble::Scoring.score('') }.must_raise ArgumentError
+      proc { Scrabble::Scoring.score('     ') }.must_raise ArgumentError
     end
   end
 
   describe 'highest_score_from' do
-    it 'returns nil if no words were passed' do
+    it 'throw ArgumentError if no words were passed' do
+      proc { Scrabble::Scoring.highest_score_from([]) }.must_raise ArgumentError
+    end
+
+    it 'throw ArgumentError if provided non-array input' do
+      proc { Scrabble::Scoring.highest_score_from(nil)}.must_raise ArgumentError
+    end
+
+    it 'throw ArgumentError if element in provided array is not a String' do
+      proc { Scrabble::Scoring.highest_score_from(["foo", 4, "bar"]) }.must_raise ArgumentError
     end
 
     it 'returns the only word in a length-1 array' do
+      Scrabble::Scoring.highest_score_from(["unicorn"]).must_equal "unicorn"
     end
 
     it 'returns the highest word if there are two words' do
+      Scrabble::Scoring.highest_score_from(["unicorn", "boo"]).must_equal "unicorn"
```

Comment:
You should also try varying the order in these tests, just to ensure it's not the order determining the winner.